index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="referrer" content="no-referrer">
+    <meta http-equiv="Content-Security-Policy" content="%VITE_CSP_META%" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <!-- WalletConnect uses the first image found in the head metadata for their wallet connect SDKs (web, kotlin, swift) -->
+    <link rel="apple-touch-icon" sizes="192x192" href="/icon-192x192.png" integrity="%VITE_SRI_ICON_192X192_PNG%" crossorigin="anonymous" />
+    <link rel="apple-touch-icon" sizes="512x512" href="/icon-512x512.png" integrity="%VITE_SRI_ICON_512X512_PNG%" crossorigin="anonymous" />
+    <link rel="icon" type="image/png" href="/favicon.png" integrity="%VITE_SRI_FAVICON_PNG%" crossorigin="anonymous" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" integrity="%VITE_SRI_FAVICON_SVG%" crossorigin="anonymous" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no,viewport-fit=cover" />
+    <meta property="og:title" content="ShapeShift DAO">
+    <meta property="og:image" content="/ss-thumb.jpg">
+    <meta property="og:type" content="website" />
+    <meta property="og:image:width" content="1200"/>
+    <meta property="og:image:height" content="630"/>
+    <meta name="twitter:card" content="summary_large_image">
+    <meta
+      name="description"
+      content="ShapeShift DAO | Your Web3 & DeFi Portal"
+    />
+    <link rel="preconnect" href="https://rsms.me/">
+    <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="/manifest.json" integrity="%VITE_SRI_MANIFEST_JSON%" crossorigin="anonymous" />
+    <link rel="prefetch" href="/env.json" as="fetch" crossorigin="anonymous" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>ShapeShift</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <script type="module" src="/src/lib/global-polyfills.ts"></script>
```

Comment:
I think we can replace with: https://github.com/davidmyersdev/vite-plugin-node-polyfills

Reply:
It doesn't work at all, I spent 2 hours trying to fix this but this plugin doesn't seems like working with our setup, can we use each polyfills during the meantime and avoid blocking this PR for this? 

I know the more dependencies we add the more risks we add but I'm pretty stuck on this right now :/

Happy to create an issue and try to ask someone else getting his eyes on it